.travis.yml

```diff
@@ -7,3 +7,5 @@ deploy:
   skip_cleanup: true
   on:
     branch: master
+after_success:
+  - npm run converage
```

Comment:
```diff
- converage
+ coverage
```

:)

Reply:
lol goodness! I should just copy paste instead of typing it out.